.github/workflows/markdown-links.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           PATTERNS: |
             **/**.md
-      - uses: gaurav-nelson/github-action-markdown-link-check@1.0.14
+      - uses: creachadair/github-action-markdown-link-check@master
```

Comment:
![](https://media.giphy.com/media/epVGITwvy07xC/giphy.gif)

Reply:
No secret here, it's a fork!